.github/issue_template.md

```diff
@@ -0,0 +1,30 @@
+## Summary
+
+<!--Summarise your issue in one sentence (what goes wrong, what did you expect to happen.)-->
+
+## Steps to reproduce
+
+<!--How can we reproduce the issue?-->
+
+## Expected behaviour
+
+<!--What did you expect the app to do?-->
+
+## Observed behaviour
+
+<!--What did you see instead?  Describe your issue in detail here.-->
+
+## Logcat for the issue
+
+<!--Provide logs for the crash here-->
+
+## Device and Android version
```

Comment:
is it necessary to take the device ? or only android version will work ?

Reply:
yeah, kinda necessary because sometimes you find different bugs in different devices and different OS versions.

I'll create an issue to add an option to write ios version as well.